src/demos/CalloutTrigger/index.tsx

```diff
@@ -0,0 +1,37 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+
+module.exports = class WithTarget extends React.PureComponent<{}, { showing: boolean }> {
+  private target: HTMLElement | null;
+
+  constructor () {
+    super();
+    this.state = {
+      showing: false,
+    };
+  }
+
+  private saveRef(ref: HTMLElement | null) {
+    this.target = ref;
+  }
+
+  private onClick = () => {
+    this.setState(({ showing }) => ({ showing: !showing }));
+  }
+
+  public render() {
+    const content = this.state.showing
+    ? React.cloneElement(this.props.children as any, { target: this.target }) : null;
```

Comment:
Is this the best that Prettier could do?

Reply:
we dont have Prettier configured for this project do we?

Reply:
Also, by doing `React.cloneElement` you're assuming that you only have a single child right? You may want to call `React.Children.only` https://reactjs.org/docs/react-api.html#reactchildrenonly

Reply:
I have Prettier configured in my editor, so I use it everywhere.

Feel free to copy the `.prettierrc` file to this repo though.